dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.1.ebuild

```diff
@@ -40,11 +40,11 @@ multilib_src_install_all() {
 	fi
 
 	if use examples ; then
-		insinto /usr/share/doc/${P}/examples
-		doins examples/daexutil.h
-		doins examples/example{0,1,2,3}.c
-		doins examples/util.c
-		doins examples/EXAMPLES.USAGE
+		insinto /usr/share/doc/${P}/Examples
```

Comment:
Gentoo does not usually keep examples in 'Examples' but 'examples'. Can you revert this change ? If the tarball includes a directory 'Examples', Gentoo can still install them in 'examples'